packages/polytechcolloscopeclient/lib/src/polytechcolloscopeclient_base.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:beautiful_soup_dart/beautiful_soup.dart';
+import 'package:html/dom.dart';
+import 'package:requests_plus/requests_plus.dart';
+
+import 'colloscope_datastructs.dart';
```

Comment:
Just import, the main export is shorter.

Reply:
Well yes but it's not a really good practice